configmaster_project/settings.py

```diff
@@ -179,5 +179,23 @@
     'etc/vmware/lunTimestamps.log',
 )
 
-# noinspection PyUnresolvedReferences
-from local_settings import *
+
+try:
+    # noinspection PyUnresolvedReferences
+    from local_settings import *
+except ImportError:
+    # No local_setting.py - running on OpenShift?
```

Comment:
I see where you're coming but, but I believe that using environment variables should be the default in absence of a settings file (it's commonly regarded as best practice). Maybe remove the except clause altogether and use as default?

Reply:
I like the idea of moving it above the local_settings import, that would be consistent with https://github.com/leoluk/configmaster/pull/38/commits/bd61ef9af0a7ee53452688765073e7f903c37c49#diff-5933c8a462128818de76fd0baeb75a00. Wouldn't this PR be the better place for these changes in settings.py? Since getting the DB config from environment variables is not specific to openshift I would rename openshift_db.py. Maybe something like db_config_from_env.py?

Reply:
> Wouldn't this PR be the better place for these changes in settings.py?

Not unless I want to spend all day rebasing my commits...